wallet/src/events.rs

```diff
@@ -66,6 +141,31 @@ pub enum EventKind {
     },
 }
 
+impl EventKind {
+    pub fn to_child(&self) -> Option<(Principal, ChildEventKind)> {
+        match *self {
+            Self::CanisterCreated { cycles, canister } => {
+                Some((canister, ChildEventKind::Created { cycles }))
+            }
+            Self::CanisterCalled {
+                canister,
+                ref method_name,
+                cycles,
+            } => Some((
+                canister,
+                ChildEventKind::Called {
+                    method_name: method_name.clone(),
+                    cycles,
+                },
+            )),
+            Self::CyclesSent { to, amount, refund } => {
+                Some((to, ChildEventKind::CyclesSent { amount, refund }))
+            }
+            _ => None,
```

Comment:
What would you think about explicitly calling out the events that are being converted to None, rather than using `_` to pick them all up?  That way when an event is added, it will be easier to notice that a decision must be made in here.

Reply:
Can't hurt.

Reply:
Can't hurt.